katello-selinux-enable

```diff
@@ -1,4 +1,7 @@
-#!/bin/bash
+#!/bin/bash    /usr/sbin/semanage port -E | \
```

Comment:
Not sure what's happening here, but it it can be removed since we got rid of elasticsearch.

Reply:
Yes remove, `foreman-selinux-disable` will also disable the port if present.